src/main/resources/db/migration/V63__add_event_logging_table.sql

```diff
@@ -0,0 +1,18 @@
+BEGIN;
+----------------------------------------
+
+DROP TABLE IF EXISTS event_logging;
+CREATE TABLE IF NOT EXISTS event_logging
+(
+    id                                      UUID        PRIMARY KEY,
+    before_data                             JSONB       DEFAULT NULL,
+    processed_data                          JSONB       DEFAULT NULL,
+    source_system_id                        TEXT        DEFAULT NULL,
+    UUID                                    UUID        DEFAULT NULL,
```

Comment:
might as well keep this lower case as well

Reply:
Is this the personKey? If so should it be a foreign key? Also should it be named person_key?